src/test/java/tests/ShopTests/ParameterizedUserDetailsTest.java

```diff
@@ -0,0 +1,51 @@
+package tests.ShopTests;
+
+import com.endava.models.Item;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import pages.*;
+import tests.BaseTest;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ParameterizedUserDetailsTest extends BaseTest {
```

Comment:
No need to use Parameterized in the class name, it is only an implementation detail. It is important for the test class name to show what would be tested inside it. Suggestion - rename it to CheckoutCredentialsTest, or similar.